packages/realm/bindgen/src/templates/node-wrapper.ts

```diff
@@ -141,6 +141,7 @@ export function generate({ rawSpec, spec: boundSpec, file }: TemplateContext): v
     "Decimal128",
     "EJSON_parse: EJSON.parse",
     "EJSON_stringify: EJSON.stringify",
+    "Symbol_for: Symbol.for",
```

Comment:
Turns out this was actually not needed on Node, since NAPI expose a method to do this.
No harm in keeping it, but its not used but the node.ts template.

Reply:
Forgot to remove that when I removed it from the node template, thanks for spotting it.